### PR TITLE
Move to relying on Titus VPC Service to send us routes

### PR DESCRIPTION
**Pull request by sargun**
This finishes wiring up jumbo frames end-to-end, but it means that
it relies on the Titus VPC Service to send down routes to the
agent with MTU-per-route.

### CODE REVIEW

File: vpc/tool/container2/setup_container_linux.go [hunk 1]
```diff
@@ -225,13 +208,13 @@ func addIPv4AddressAndRoute(ctx context.Context, nsHandle *netlink.Handle, link
 	err = nsHandle.RouteDel(&oldLinkRoute)
 	if err != nil {
 		logger.G(ctx).WithError(err).Error("Unable to delete route on link")
-		return errors.Wrap(err, "Unable to add delete existing 'link' route to link")
+		return 0, errors.Wrap(err, "Unable to add delete existing 'link' route to link")
 	}
 
-	return nil
+	return mtu, nil
```
Comment (solarkennedy) [inline vpc/tool/container2/setup_container_linux.go:214]: This return of mtu from this function confuses me a little bit.
I get it though, this is exactly where the actual default route mtu is calculated, and needs to be consumed later on.

But would a function called `getDefaultRouteMtu()` (to be injected into setupHTBClasses) that does this make the code any simpler?

File: executor/runtime/docker/docker.go [hunk 2]
```diff
@@ -1881,13 +1886,12 @@ func setupNetworkingArgs(burst bool, c *runtimeTypes.Container) []string {
...
-	if jumbo, ok := c.TitusInfo.GetPassthroughAttributes()[jumboFrameParam]; ok {
-		if val, err := strconv.ParseBool(jumbo); err != nil {
-			log.Error("Could not parse value for "+jumboFrameParam+": ", err)
-		} else if val {
-			args = append(args, "--jumbo=true")
-		}
+	if jumbo, err := c.Jumbo(); err != nil {
+		log.WithError(err).Error("Could not parse value for jumbo frame parameter")
+	} else if jumbo {
+		args = append(args, "--jumbo=true")
```
Comment (solarkennedy) [inline executor/runtime/docker/docker.go:1892]: So we have to have both things in place here:
* task attribute to allow jumbo
* New version of vpc service that sends routes with mtus

Anything else?

I'm trying to wrap my head around deployment strategy and potential failure modes between the components.

**Review by solarkennedy — APPROVED** [review]

File: vpc/tool/container2/setup_container_linux.go (hunk 1 above)
Comment (sargun) [reply vpc/tool/container2/setup_container_linux.go:214]: I feel like then we're writing the code to see if a default route exists & returning the MTU in two different places, and that seems a little weird to me?

File: executor/runtime/docker/docker.go (hunk 2 above)
Comment (sargun) [reply executor/runtime/docker/docker.go:1892]: The task attribute to allow jumbo has been a thing for a long time, but yeah, the thing that goes out "first" now is the VPC service that sends down routes. Earlier this was written with the idea that the agent would go out first before the VPC Service was taught to send down any routes.

**Pull request merged**